### PR TITLE
Feat: Apply form Added

### LINKED ISSUE #24: Apply form for the Code For Govtech
## Description
Develop an apply form to collect information from individuals interested in contributing to CodeforGovtech's open-source projects. The form should include the following questions:
- Basic Details:
  - Name (Text field)
  - College/Organisation Name (Text field)
  - Degree (Text field)
  - Email (Email field)
  - Phone Number (Text field)
- Discord and GitHub ID:
  - Discord ID (Text field, optional)
  - GitHub ID (Text field, optional)
- Experience in Open-Source:

## Goals
Create an apply form for CodeforGovtech to collect basic details (Name, College/Organisation name, Degree, email, phone number), Discord and GitHub IDs (optional), and assess experience in open-source (Yes/No).

## Expected Outcome
To have a form to apply

## Mockups / Wireframes
N/A

## Project
Code for GovTech

## Tech Skills Needed:
Next.js

## Mentor(s)
@Shruti3004 

## Complexity
Low

## Category
UI/UX

### PR DESCRIPTION
**Pull request by homosapien14**
Describe:
added a feature to get user's basic details via form.

Fixes #24

### CODE REVIEW

**Review by void-ness — COMMENTED** [review]
Hey, The overall form looks good. I noticed few areas which can be improved and would love to know your opinion about the same.

File: src/components/form/index.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+      github,
+      openSourceExp,
+    };
+    localStorage.setItem("draftData", JSON.stringify(draftData));
+  };
+
+  return (
+    <>
+      <Header />
+      <div className="container w-3/5 lg:w-2/3 z-30 mx-auto mt-6 lg:-mt-6 mb-16">
```
[the remark this answers is not in the document]
Comment (homosapien14) [reply src/components/form/index.tsx:47]: hey, @void-ness can you please tell me the screen size to check it in my device?

File: src/components/form/index.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from "react";
+import InputField from "./inputField";
+import Header from "./header";
+
+export const Form: React.FC = () => {
+  const [name, setName] = useState("");
```
[the remark this answers is not in the document]
Comment (homosapien14) [reply src/components/form/index.tsx]: Hey, thank you for noticing it. I missed this. I have changed it here: [`73cfb14`](https://github.com/Code4GovTech/website/pull/23/commits/73cfb14a1f9c2620c1817d840ce00b82eb878e50)

File: src/components/form/header.tsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (homosapien14) [reply src/components/form/header.tsx]: resolved this, make component name to FormHeader.

File: src/components/form/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (void-ness) [reply src/components/form/index.tsx:38]: It is happening for devices under the width of 400px

**Comment by homosapien14** [comment]
hey @Shruti3004 any updates on this feature?

**Pull request merged**